package.json

```diff
@@ -5,7 +5,8 @@
   "author": "David Obodo <obododavid5@gmail.com>",
   "license": "MIT",
   "scripts": {
-    "build": "webpack --mode production",
+    "build": "NODE_ENV=production webpack",
```

Comment:
Is your build script different from the one i wrote, because i believe they mean the same thing, or do they not?